server/routerlicious/packages/memory-orderer/src/localOrderer.ts

```diff
@@ -355,7 +368,7 @@ export class LocalOrderer implements IOrderer {
 			this.tenantId,
 			this.documentId,
 			this.gitManager,
-			null /* deltaService */,
+			null as unknown as IDeltaService /* deltaService */,
```

Comment:
Should this paramater to `SummaryReader` be made to allow null? These casts are effectively working around the type-safety this PR is aimed to add.

Reply:
Fair point, I can address that